### DISCUSSION TITLE
Forgetting curve clock

### DISCUSSION THREAD

**Discussion by ghost** [you-apps/ClockYou#366] [Ideas]
### Describe your suggested feature

Generate a clock based on the forgetting curve

1. clock Title
2. Start/End button
3. Generate Forgetting curve clock after clicking End

forgetting curve Scientifically arrange time to review knowledge 

### Other details

https://en.wikipedia.org/wiki/Forgetting_curve

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Sorry, that's not planned, external PRs would be welcome though.

**Comment by ghost** [comment]
This is an interesting feature
